static/src/javascripts/projects/common/modules/ui/welcomeBanner.js

```diff
@@ -0,0 +1,48 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/utils/load-css-promise'
+], function (
+    fastdom,
+    $,
+    storage,
+    template,
+    loadCssPromise
+) {
+    /**
+     * Rules:
+     * 1st visit replace nav bar with banner
+     */
+    var header = document.getElementById('header'),
+        message1 = '<div class="banner-message"><%=HTML%></div>',
+        DATA = {
+            MESSAGE1: {
+                HTML: '<span class="pull-left">news website</span><span class="pull-right">of the year</span>'
+            }
+        };
+
+    function showWelcomeMessage(messageNumber) {
```

Comment:
We can decide that when we'll actually have all the messages. At the moment nothing in this file is used; soon after merging this I will create a new PR with the A/B tests for the messages.